fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -184,6 +192,13 @@ public long getDate() {
         }
     }
 
+    /**
+     * Get the event ID.
+    **/
+    public String getEventID() {
```

Comment:
As a note, there is already a `getIdentifier()` method on this class... which appears to be only used in test code. It would be nice to remove the ambiguity of two get-ID methods if possible.